pkg/sync/v6/data/PrometheusRule.monitoring.coreos.com/openshift-monitoring/prometheus-k8s-arorules.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: prometheus-k8s-arorules
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: aro-sync.rules
+    rules:
+    - alert: SyncPodDown
+      annotations:
+        description: Sync Pod is down
+        summary: Sync Pod is down
```

Comment:
It would be really useful if we could add a `dashboard` entry (with link to a dashboard for the service) however I think this can be done on the geneva side albeit "manually" :man_shrugging:

Reply:
I want to keep the abstraction layers clean, so no Geneva specific stuff in Prometheus.
Especially since the link will be different for staging and prod.